### ISSUE TITLE
Duplicated settings

### ISSUE THREAD

**Issue by Falc** [dmishh/SettingsBundle#28]
I have found a weird problem when using alpha2, the database is storing duplicated settings.

<hr>

With this configuration...
```yaml
dmishh_settings:
    settings:
        test1: ~
```

...and this code in a Symfony command:
```php
protected function execute(InputInterface $input, OutputInterface $output)
{
    $manager = $this->getContainer()->get('settings_manager');

    $manager->set('test1', 1);
    $manager->set('test1', 2);
    $manager->set('test1', 3);
}
```

Everything works fine, the "dmishh_settings" table stores a single row:

| id | name  | value | ownerId |
|----|-------|-------|---------|
| 1  | test1 | i:3;  | NULL    |

<hr>

With this configuration...
```yaml
dmishh_settings:
    settings:
        test1: ~
        test2: ~
```

...and this code in a Symfony command:
```php
protected function execute(InputInterface $input, OutputInterface $output)
{
    $manager = $this->getContainer()->get('settings_manager');

    $manager->set('test1', 1);
    $manager->set('test1', 2);
    $manager->set('test1', 3);
    $manager->set('test2', true);
}
```

It inserts 5 rows in "dmishh_settings":

| id | name  | value | ownerId |
|----|-------|-------|---------|
| 1  | test1 | i:3;  | NULL    |
| 2  | test2 | N;    | NULL    |
| 3  | test2 | N;    | NULL    |
| 4  | test2 | b:1;  | NULL    |
| 5  | test1 | i:3;  | NULL    |

**Comment by dmishh** [comment]
Thanks, @Falc 
I'll check this out soon

**Comment by dmishh** [comment]
Fixed in 6c88cd6
Thanks to @Nyholm :+1:
